.github/workflows/auto-approve.yml

```
@@ -1,7 +1,9 @@
 # Approve PRs with "pr/auto-approve". mergify takes care of the actual merge.
 
 name: auto-approve
-on: pull_request
+on:
+  pull_request:
+    types: [ labeled ]
```

Per [docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request) -

> By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened

Should we include at least these and maybe some others.


```suggestion
    types: [ labeled, unlabeled, opened, synchronize, reopened ]
```

```suggestion
    types: [ labeled, unlabeled, opened, synchronize, reopened, ready_for_review, review_requested ]
```

---

This is the list I had used previously - https://github.com/aws/aws-cdk/blob/6de792c3453cec17a9c4b0fca47477d69e5ed36e/.github/workflows/v2-pull-request.yml#L7 - and they worked fine. There were some duplicate approvals, but I would lean that way rather than missed approvals.